apps/web/src/app/[orgShortcode]/convo/_components/create-convo-form.tsx

```diff
@@ -306,9 +344,29 @@ export default function CreateConvoForm({
     newEmailParticipants
   ]);
 
-  const [editorText, setEditorText] = useState<JSONContent>(
-    emptyTiptapEditorContent
-  );
+  const [editorText, setEditorText] = useState(draft.content);
+
+  // Autosave draft
+  const debouncedEditorText = useDebounce(editorText, 500);
```

Comment:
this debounce can be much higher
5 seconds even

Reply:
that would have the issue of not saving your editor text of last 5 second if you navigate early